book.rb

```diff
@@ -0,0 +1,14 @@
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+
+  def add_rental(rental)
+    rentals << rental
+  end
```

Comment:
- [x] This approach lacks clear associations between Person, Book, and Rental, making it challenging to maintain a many-to-many relationship. I recommend considering the below suggestion, which encapsulates rental creation within the Book class and properly sets associations, improving code clarity and maintainability. 🔄
```suggestion
 def add_rental(person, date)
    `# *the book itself should be sent as a parameter to create the new rental achieving the 'has-many' association.`
    Rental.new(date, self, person)
  end
```